Tests/LibCompress/TestDeflate.cpp

```diff
@@ -155,3 +156,17 @@ TEST_CASE(deflate_compress_literals)
     Array<u8, 0x13> test { 0, 0, 0, 0, 0x72, 0, 0, 0xee, 0, 0, 0, 0x26, 0, 0, 0, 0x28, 0, 0, 0x72 };
     auto compressed = TRY_OR_FAIL(Compress::DeflateCompressor::compress_all(test, Compress::DeflateCompressor::CompressionLevel::GOOD));
 }
+
+TEST_CASE(ossfuzz_63183)
+{
+    // This makes sure that the tests will run both on target and in Lagom.
+#ifdef AK_OS_SERENITY
```

Comment:
Many other files have a somewhat standardized macro for this, see eg LibGfx image loading tests or LibGL tests, iirc

Reply:
Cool, changed this to match them.